coriolis/data_transfer.py

```diff
@@ -24,6 +24,9 @@
                     'will be done through this service. This value can be '
                     'either a unix socket path (/var/run/compressor.sock '
                     'or an IP:PORT.'),
+    cfg.IntOpt('default_requests_timeout',
```

Comment:
There is no need to register the option multiple times, only once. (unless it's in separate option groups (i.e. you pass `CONF.register_opts(<opts>, "OPTION_GROUP_NAME")`)

Please define this in a separate `coriolis/conf.py` module, and simply make sure that you run a `from coriolis import conf` inside the `__init__.py` of the `coriolis.cmd` package, since all the entry points of the projects are located there.